src/services/mentees.js

```diff
@@ -1081,6 +1083,11 @@ module.exports = class MenteesHelper {
 			// update count after filters
 			userDetails.data.result.count = userDetails.data.result.count
 
+			// add index number to the response
+			userDetails.data.result.data = userDetails.data.result.data.map((data, index) => ({
```

Comment:
@adithyadinesh0412 please update  API response  and if require in postman collection